### DISCUSSION TITLE
Проблема с загрузкой текстур

### DISCUSSION THREAD

**Discussion by SaberShoker** [PavelBlend/blender-xray#382] [Q&A] [answered]
При импорте объекта, не отображается текстура во вьюпорте, но на UV развертке и Texture Paint текстура отображается корректно, с чем может быть связанна проблема?

**Comment by SaberShoker** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/69687233/120541072-752c2600-c3fa-11eb-9c20-a630572ce0c6.png)
Выглядит это вот так

**Reply by PavelBlend** [reply]
@SaberShoker а какой режим отображения включён? Есть два способа включить текстуры:
![01](https://user-images.githubusercontent.com/7983249/121590236-c498d480-ca40-11eb-8cdc-49dcfd210538.jpg)
![02](https://user-images.githubusercontent.com/7983249/121590251-c9f61f00-ca40-11eb-8564-8750547b08bb.jpg)

Или если активирован рендер EEVEE вместо Cycles, то можно включить режим рендера.

**Reply by SaberShoker** [reply]
Спасибо разобрался
